Resources/Prototypes/Entities/Structures/Machines/bombs.yml

```diff
@@ -25,8 +25,8 @@
     # If you nerf the syndicate bomb in any major way, this should probably drop down to at least 100s (not 90s to compensate for slower movement speed & less lag in SS14)
     # Unless, of course, you want the 90 seconds regardless. I can't stop you.
     - type: OnUseTimerTrigger
-      delay: 120
-      delayOptions: [120, 150, 180, 210, 240, 270, 300]
+      delay: 300
```

Comment:
Default delay should be set to the lowest option (180) as it is for every other bomb.

Reply:
Skill issue if user can't set the delay lower

Reply:
Every other bomb in the game works like this. Most people aren't going to check the timer on the massive bomb they just bought. It should be set to the lowest value by default.

Reply:
Every other bomb in the game is less destructive and has a smaller boom. Defaults matter and I think this is reasonable if you are not willing to compromise on minimum timer setting.

Reply:
I don't see how this would reasonably balance it. if someone wanted to have 180 seconds, they easily can. This just means people who don't realize there's a shorter timer option (which will likely be many, because of no CL) won't use it- and I feel saying "most people won't bother to shorten the timer" is a bad way to balance it when people can, with one button, almost halve the timer.